platform/qemu.go

```diff
@@ -395,7 +396,8 @@ func setupIgnition(confPath string, diskImagePath string) error {
 	// Using libvirt can lead to permission denied issues if it does not have access
 	// rights to the qcow image
 	os.Setenv("LIBGUESTFS_BACKEND", "direct")
-	cmd := exec.Command("guestfish", "--listen", "-a", diskImagePath)
+	cmd := exec.Command("guestfish", "--listen", "--keys-from-stdin", "--key", "/dev/sda4:file:/dev/null", "-a", diskImagePath)
```

Comment:
Is hardcoding `/dev/sda4` correct?

Reply:
Yeah, see https://github.com/coreos/coreos-assembler/blob/92a8ee879b8a8cb94981e679b92be9af6c19158c/src/create_disk.sh#L88. A more robust approach though would be to look for the root label. But let's not write that code before discussing some way to share this between cosa and mantle. :)

Reply:
I can pass it in to the  `setupIgnition` function , but i guess it needs to be passed in someplace. the coreos-assembler has the root partition hardcoded as 4, so I guess we should be fine unless that changes.

Reply:
@jlebon yes, my knowledge of guestfish is limited but i was also wondering if there is a way to do that..